src/tribler/gui/widgets/channeldescriptionwidget.py

```diff
@@ -225,7 +225,7 @@ def show_description_page(self):
 
     def _on_description_received(self, result):
         if result:
-            self.description_text = result["description_text"]
+            self.description_text = result.get("description_text", None)
```

Comment:
From the previous version of the code, it looks like the code assumes that the `description_text` value can be `None`, but the `description_text_preview` value is always a string. After the fix, we set `description_text_preview` to `None` as well. I don't know whether it is correct or not, but apparently, it works.

I think we can simplify the code a bit.

First, the `get` method of the dictionary uses `None` as the default value for the second argument, so it is unnecessary to specify it.

Second, the two branches of the `if` statement look similar, so instead of
```python
     if result:
            self.description_text = result.get("description_text")
            self.description_text_preview.setMarkdown(self.description_text)
        else:
            self.description_text = None
            self.description_text_preview.setMarkdown("")
```
we can write something like
```python
        self.description_text = result.get("description_text") if result else None
        self.description_text_preview.setMarkdown(self.description_text or "")
```